no-discourse-topic-required.txt

```diff
@@ -0,0 +1,4 @@
+docs/index.rst
```

Comment:
In one of your DMs, you had a much bigger list (on the order of tens of topics), was that a false alarm or do I misunderstand something?